packages/dcos-integration-test/extra/test_mesos.py

```diff
@@ -137,6 +137,7 @@ def _post_agent(url, headers, json=None, data=None, stream=False):
 
 def test_files_api(dcos_api_session):
     app, test_uuid = test_helpers.marathon_test_app()
+    app['cmd'] = 'echo $DCOS_TEST_UUID && ' + app['cmd']
```

Comment:
It looks like github does not allow me to comment on the lines below, so I'm leaving the comment here. For an outside observer it is not directly clear what this test is doing. As discussed online, please add a comment either at the beginning of the test or below, explaining that:
* the test basically checks that both stdout and stderr are not empty;
* Mesos is expected to write to stderr only.